tests/unittests/audio/test_sdr.py

```diff
@@ -34,12 +34,12 @@
 Input = namedtuple("Input", ["preds", "target"])
 
 inputs_1spk = Input(
-    preds=torch.rand(4, 1, 1, 1000),
-    target=torch.rand(4, 1, 1, 1000),
+    preds=torch.rand(2, 1, 1, 500),
+    target=torch.rand(2, 1, 1, 500),
 )
 inputs_2spk = Input(
-    preds=torch.rand(4, 1, 2, 1000),
-    target=torch.rand(4, 1, 2, 1000),
+    preds=torch.rand(2, 1, 2, 500),
+    target=torch.rand(2, 1, 2, 500),
```

Comment:
How about even 200?

Reply:
Nope, there seems to be a lower bound for the `time` dimension for this metric.
I tried many different values.